tembo-operator/src/cloudnativepg/hibernate.rs

```diff
@@ -145,6 +146,23 @@ pub async fn reconcile_cluster_hibernation(cdb: &CoreDB, ctx: &Arc<Context>) ->
         }
     }
 
+    // Remove IngressRouteTCP route for stopped instances
+    let ingress_route_tcp_name = format!("{}-ro-0", cdb.name_any().as_str());
+    if let Err(err) = delete_ingress_route_tcp(
+        Api::namespaced(client, &namespace),
```

Comment:
I don't think this will work.  It will need to be typed, so maybe adding something like this will work better

```
let ingress_route_tcp_api: Api<IngressRouteTCP> = Api::namespaced(ctx.client.clone(), namespace);
```

Then just passing in `ingress_route_tcp_api` to the `delete_ingress_route_function`